tests/test_dst_components.py

```diff
@@ -0,0 +1,350 @@
+# SPDX-FileCopyrightText: ASSUME Developers
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import pandas as pd
+import pyomo.environ as pyo
+import pytest
+
+from assume.units.dst_components import (
+    DriPlant,
+    DRIStorage,
+    ElectricArcFurnace,
+    Electrolyser,
+    GenericStorage,
+)
+
+
+# Test Electrolyser class
+@pytest.fixture
```

Comment:
many tests had the same name, which doesn't work. I fixed it, but please use different names in the future

Reply:
After a good thought, I think we can remove the test for dst_components for individual technologies sicne we have tested them in the test for the steel plant, already. What's your opinion on this?

Reply:
it is up to you, what you think would be best for future development. But I believe keeping individual tests is also good since maybe these components will be reused in something else.

Reply:
Okay, then we will kept the test script for dst_components. :)

Reply:
@nick-harder, @maurerle I have enhanced the test script for the dst_components to ensure it thoroughly tests the functionality and constraints of the individual components. Meanwhile, I have only written the test for the electrolyser. If you give a green signal I will proceed writing similar tests for the rest of the components. The following updates were made:

    Added a dummy objective function to minimize operating costs, encouraging the model to optimize power input values.

    Included detailed checks for ramp-up and ramp-down constraints to verify that the electrolyser respects these constraints at each time step.
    
    Ensured that the power input remains within the specified minimum and maximum bounds.